cmake/tests.cmake

```diff
@@ -214,4 +214,5 @@ target_link_libraries(lite-arena-test libprotobuf-lite gmock_main)
 
 add_custom_target(check
   COMMAND tests
+  DEPENDS tests test_plugin
```

Comment:
I thought a little more and probably agree with your decision.
I added this command as an alias for compatibility with similar check for autotools project, but I forgot to specify dependencies.